tests/integration/components/sl-chart-test.js

```diff
@@ -0,0 +1,117 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-chart', 'Integration | Component | sl chart', {
+    integration: true
+});
+
+test( 'Defaults applied correctly', function( assert ) {
+    this.set( 'testseries', []);
+    this.set( 'testoptions', {});
+
+    this.render( hbs`
+      {{#sl-chart series=testseries options=testoptions}}
+      {{/sl-chart}}
+    ` );
+
+    assert.ok(
+      this.$( '.sl-chart' ).hasClass( 'sl-chart' ),
+      'has class sl-chart'
+    );
+
+    assert.ok(
+      this.$( '.sl-chart' ).hasClass( 'panel' ),
+      'has class panel'
+    );
+
+    assert.ok(
+      this.$( '.sl-chart' ).hasClass( 'panel-default' ),
+      'has class panel-default'
+    );
+
+    assert.ok(
+      this.$( '.sl-chart' ).hasClass( 'sl-panel' ),
+      'has class sl-panel'
+    );
+});
+
+test( 'Loading state adds loading class', function( assert ) {
+    this.set( 'testseries', []);
+    this.set( 'testoptions', {});
+
+    this.render( hbs`
+      {{#sl-chart series=testseries isLoading=false options=testoptions}}
+      {{/sl-chart}}
+    ` );
+
+    assert.strictEqual(
+        this.$( '.sl-chart' ).hasClass( 'sl-loading' ),
+        false,
+        'Default rendered component does not have "sl-loading" class'
+    );
+
+    this.render( hbs`
+      {{#sl-chart series=testseries isLoading=true options=testoptions}}
+      {{/sl-chart}}
+    ` );
+
+    assert.strictEqual(
+        this.$( '.sl-chart' ).hasClass( 'sl-loading' ),
+        true,
+        'Default rendered component does have "sl-loading" class'
+    );
+});
+
+test( 'Title property is set', function( assert ) {
+    this.set( 'testseries', []);
+    this.set( 'testoptions', {});
+
+    this.render( hbs`
+      {{#sl-chart series=testseries options=testoptions}}
+      {{/sl-chart}}
+    ` );
+
+    assert.strictEqual(
+        this.$( '.panel-heading' )[0],
```

Comment:
Why is this accessed via `[0]` whereas the assertion further down can be accessed via `html()` without the array index?